spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -516,6 +516,79 @@
         end
       end
 
+      describe "Line 10 exemptions" do
+        context "0 qualified dependent children" do
+          let(:submission) {
+            create :efile_submission, tax_return: nil, data_source: create(
+              :state_file_nj_intake
+            )
+          }
+          it "does not fill in" do
+            expect(pdf_fields["Text47"]).to eq ""
+            expect(pdf_fields["undefined_12"]).to eq ""
+            expect(pdf_fields["x  1500"]).to eq ""
+          end
+        end
+
+        context "1 qualified dependent child" do
+          let(:submission) {
+            create :efile_submission, tax_return: nil, data_source: create(
+              :state_file_nj_intake,
+              :df_data_two_deps,
+            )
+          }
+          it "fills in properly" do
```

Comment:
[dust] this test name could be more descriptive. Maybe "fills in number and exemption value" or something

Reply:
good idea, i'll update